### ISSUE TITLE
annotation files must be perfectly aligned

### ISSUE THREAD

**Issue by RyanLongchamps** [omerwe/polyfun#94]
Hi Omer,

For the below command I am getting the resulting error, could you please clarify what it is trying to tell me?

```
python3 polyfun.py --compute-h2-L2 \
    --output-prefix output/multi_annot.test/testb. \
    --sumstats output/multi_annot.test/testb.parquet \
    --ref-ld-chr test.set/test.set.,test.set.b/test.set.b. \
    --w-ld-chr test.set/test.set.weights.
```

```
[INFO]  Computing per-SNP h^2 for each chromosome...
  0%|                                                                                                      | 0/22 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "./polyfun/polyfun.py", line 852, in <module>
    polyfun_obj.polyfun_main(args)
  File "./polyfun/polyfun.py", line 776, in polyfun_main
    self.polyfun_h2_L2(args)
  File "./polyfun/polyfun.py", line 601, in polyfun_h2_L2
    self.compute_snpvar(args, use_ridge=True)
  File "./polyfun/polyfun.py", line 352, in compute_snpvar
    df_snpvar_chr = self.compute_snpvar_chr(args, chr_num, use_ridge=use_ridge)
  File "./polyfun/polyfun.py", line 301, in compute_snpvar_chr
    df_annot_chr = self.load_annotations_file(args, chr_num, use_ridge)
  File "./polyfun/polyfun.py", line 264, in load_annotations_file
    raise ValueError('Different annotation files of chromosome %d must be perfectly aligned'%(chr_num))
ValueError: Different annotation files of chromosome 1 must be perfectly aligned
```

My assumption is that it is telling me that the SNPs for the 2 annotation files I am trying to run L2-regularized S-LDSC with are not in the same order across files? However, I can confirm this is not the case, additionally there are no allele flips or any other obvious issues across the files either. Happy to provide file examples over email if that would assist.

Thank you!

**Comment by omerwe** [comment]
Hi @RyanLongchamps, please send me a minimal example file to help replicate this problem (oweissbrod@hsph.harvard.edu). Thanks!

**Comment by RyanLongchamps** [comment]
Hi Omer,

I was able to determine the issue. My test.set.b annotation files were in the `.annot.gz` format. I then performed all of my filtering and output them as a parquet file without ever having removed the original `annot.gz` files.

My command was picking up the `annot.gz` files (same filename prefix), instead of the `.annot.parquet` files, thus resulting in the script seeing them as different lengths.
